engine/apps/mobile_app/urls.py

```diff
@@ -14,7 +12,6 @@
     optional_slash_path("auth_token", MobileAppAuthTokenAPIView.as_view(), name="auth_token"),
 ]
 
-if settings.FCM_RELAY_ENABLED:
-    urlpatterns += [
-        optional_slash_path("fcm_relay", FCMRelayView.as_view(), name="fcm_relay"),
-    ]
+urlpatterns += [
```

Comment:
shouldn't this endpoint should be disabled for OSS setups?

Reply:
This `FCM_RELAY_ENABLED` check is now here: https://github.com/grafana/oncall/pull/1132/files#diff-b163a51eb0fb7ccba294fc4366d56d9b3343cd04ec41d51cc6dad62a5dc0f1fcR37, it seems to me like it's easier to understand this way (also easier to enable/disable it when testing)